src/main/electron-context-menu.ts

```diff
@@ -25,28 +25,52 @@ import { tx } from './load-translations'
 
 type todo = any
 
+type NotDeletedMenuItems =
+  | electron.MenuItemConstructorOptions
+  | electron.MenuItem
+
 const webContents = (win: BrowserWindow) => win.webContents
 
 const removeUnusedMenuItems = (
-  menuTemplate: (electron.MenuItemConstructorOptions | electron.MenuItem)[]
-) => {
-  let notDeletedPreviousElement:
+  menuTemplate: (
     | electron.MenuItemConstructorOptions
     | electron.MenuItem
+    | boolean
```

Comment:
might make sense to replace 
```js
   | electron.MenuItemConstructorOptions
   | electron.MenuItem
```
by
```js
| NotDeletedMenuItems
```

what do you think?

Reply:
Think I found a better name and approach now!